pom.xml

```diff
@@ -174,18 +166,15 @@
     <jsendnsca.version>2.0.1</jsendnsca.version>
     <neethi.version>2.0.1</neethi.version>
     <barbecue.version>1.5-beta1</barbecue.version>
-    <jaxws-spring.version>1.8</jaxws-spring.version>
     <antlr-complete.version>3.5.2</antlr-complete.version>
-    <hibernate-core.version>5.4.24.Final</hibernate-core.version>
     <license.inception.year>2002</license.inception.year>
-    <jsr181-api.version>1.0-MR1</jsr181-api.version>
     <ascsapjco3wrp.version>20100529</ascsapjco3wrp.version>
     <testng.version>6.5.2</testng.version>
     <cglib.version>2.2</cglib.version>
     <maven-failsafe-plugin.reuseForks>false</maven-failsafe-plugin.reuseForks>
     <jt400.version>6.1</jt400.version>
     <sac.version>1.3</sac.version>
-    <hibernate-jpa-2.1-api.version>1.0.2.Final</hibernate-jpa-2.1-api.version>
+    <quartz.version>1.7.2</quartz.version>
```

Comment:
Is this correct?
Under [BACKLOG-41266](https://hv-eng.atlassian.net/browse/BACKLOG-41266) we moved from 1.7.2 to 2.3.2!